src/app/pages/send-tokens/components/send-form-inner.tsx

```diff
@@ -67,6 +71,44 @@ export function SendFormInner(props: SendFormInnerProps) {
     values.recipient,
   ]);
 
+  useEffect(() => {
+    void fetchMandatoryMemoAddresses();
+  }, []);
+
+  useEffect(() => {
+    if (mandatoryMemoAddresses.includes(values.recipient)) {
+      setIsMemoRequired(true);
+      return;
+    }
+    setIsMemoRequired(false);
+  }, [values.recipient, mandatoryMemoAddresses]);
```

Comment:
This should use the validation functionality of formik, declared in the upper component

Reply:
done